CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+# Contributing
+
+A little check list of contributions
+
+ - Create an issue if you want add a bigger feature and lets talk about it
+ - Take a look at the documentation [development.md](docs/development.md)
+ - Take a look at the [pull request template](.github/PULL_REQUEST_TEMPLATE.md)
+ - If you need help don't hesitate to ask questions in pull requests or issues
+ 
+ 
+If you don't want to add more tests I would also love to see pull requests or issues with your
+custom fork and implementations.
```

Comment:
Maybe a comma would improve the readability?

... more tests**,** I would also ...

Reply:
Good point, removed it and replaced it with `Even if your code is not perfect I...` to encourage contributions.